src/pages/Landing/Landing.scss

```
@@ -0,0 +1,389 @@
+* {
+  box-sizing: border-box;
+  color: white;
+}
+
+.view {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.MainTitle {
+  color: #d2c030;
+  font-weight: 500;
+  font-size: 14px;
+  @media screen and (min-width: 768px) {
```

반복되는 너비는 변수로 사용해도 좋을 것 같아요

scss 파일 상단에서 `@import "/src/styles/mixin_var.scss";` 추가하고, 아래처럼 사용하실 수 있습니다!
```suggestion
  @media screen and (min-width: $tablet) {
```